src/app/project/manage-payments/revenue-share-confirm-modal/revenue-share-confirm-modal.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RevenueShareConfirmModalComponent } from './revenue-share-confirm-modal.component';
+
+describe('RevenueShareConfirmModalComponent', () => {
+  let component: RevenueShareConfirmModalComponent;
+  let fixture: ComponentFixture<RevenueShareConfirmModalComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ RevenueShareConfirmModalComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RevenueShareConfirmModalComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
the tests are failing because of this file. Remove `*.spec.ts` files if you don't write tests.